src/config/2021.json

```
@@ -1101,7 +1101,8 @@
       "teams": [
         "reviewers"
       ],
-      "avatar_url": "83400777",
+      "avatar_url": "83400777", 
+       "website": "https://www.paulteitelman.com/",
```

Looks like one too many spaces:

```suggestion
      "website": "https://www.paulteitelman.com/",
```